vars/saveCache.groovy

```diff
@@ -23,7 +24,7 @@ def call(String cache_file, String directory, Integer clean_old = 0, String excl
             """            
         }
         if (clean_old > 0){
-            sh "find $HOME/cache/${URLDecoder.decode(JOB_NAME)} -name '*.tar.gz' -ctime +${clean_old} -delete"
+            sh "find $HOME/cache/${jobName} -name '*.tar.gz' -ctime +${clean_old} -delete"
```

Comment:
Does this work both for PR and branches?

Reply:
Let me try

Reply:
Yes, it does @fchavant